src/Custom-sample.h

```diff
@@ -424,49 +424,49 @@ static const char DATA_ESPEASY_DEFAULT_MIN_CSS[] PROGMEM = {
 // #define USES_P070   // NeoPixel_Clock
 // #define USES_P071   // Kamstrup401
 // #define USES_P072   // HDC1000/HDC1008/HDC1010/HDC1050/HDC1080
-// #define USES_P073   // 7DG
+// #define USES_P073   // 7-segment display
 // #define USES_P074   // TSL2591
 // #define USES_P075   // Nextion
-// #define USES_P076   // HWL8012   in POW r1
-// #define USES_P077   // CSE7766   in POW R2
-// #define USES_P078   // Eastron Modbus Energy meters
-// #define USES_P079   // Wemos Motoshield
+// #define USES_P076   // HLW8012/BL0937
+// #define USES_P077   // CSE7766
+// #define USES_P078   // Eastron SDMxxx Modbus
+// #define USES_P079   // Wemos / Lolin Motorshield
 
 // #define USES_P080   // iButton Sensor  DS1990A
 // #define USES_P081   // Cron
 // #define USES_P082   // GPS
-// #define USES_P083   // SGP30
+// #define USES_P083   // SGP30 TVOC/eCO2
```

Comment:
I know the vendors are really eager to mention "eCO2" with these VOC sensors, but it is really confusing as these sensors absolutely do not measure CO2.
So either please add "(not CO2)" or remove "eCO2".